src/App.js

```diff
@@ -1,25 +1,86 @@
-import logo from './logo.svg';
-import './App.css';
+//import { Routes, Route } from "react-router-dom";
+import styled from "styled-components";
+import { useEffect, useState } from "react";
+import { getFromLocal, setToLocal } from "../src/lib/localStorage";
+import SearchForm from "./components/SearchForm";
+import SearchList from "./components/SearchList";
+import ShoppingList from "./components/ShoppingList";
+import LanguageOption from "./components/LanguageOpiton";
+
+/* import { useImmer } from "use-immer"; */
 
 function App() {
+  const [allItems, setAllItems] = useState([]);
+  const [searchItems, setSearchItems] = useState([]);
+  const [inputValue, setInputValue] = useState("");
```

Comment:
Just a thought: personally I would name it a bit differently `const [searchValue, setSearchValue] = useState("");`. Just to be a bit more clear and direct about what this input value is supposed to be.